xclim/indices/stats.py

```diff
@@ -485,16 +486,29 @@ def _fit_start(x, dist, **fitkwargs) -> tuple[tuple, dict]:
         scale = ((x - loc) ** chat).mean() ** (1 / chat)
         return (chat,), {"loc": loc, "scale": scale}
 
-    if dist == "gamma":
+    if dist in ["gamma"]:
         x_pos = x[x > 0]
-        m_pos = x_pos.mean()
-        log_of_mean = np.log(m_pos)
+        m = x_pos.mean()
+        log_of_mean = np.log(m)
         mean_of_logs = np.log(x_pos).mean()
         a = log_of_mean - mean_of_logs
         alpha = (1 + np.sqrt(1 + 4 * a / 3)) / (4 * a)
-        beta = m_pos / alpha
+        beta = m / alpha
         return (alpha,), {"scale": beta}
 
+    if dist in ["fisk"]:
+        x_pos = x[x > 0]
+        m = x_pos.mean()
+        v = x_pos.var()
+        # pdf =  (beta/alpha) (x/alpha)^{beta-1}/ (1+(x/alpha)^{beta})^2
+        # Compute f_1 and f_2 which only depend on beta:
+        # f_1 := mean/alpha = <x>/alpha
+        # f_2 := variance/alpha^2 = (<x^2> - <x>^2)/alpha^2
+        # In the large beta limit, f_1 -> 1 and f_1/sqrt(f_2) -> 0.56*beta - 0.25
+        # Solve for alpha and beta below:
+        alpha_0, beta_0 = m, (1 / 0.56) * (m / np.sqrt(v) + 1 / 4)
```

Comment:
I didn't find any parameter estimation in the literature, so I computed my own. I described the process, but should I include more details / include a notebook somewhere?

For reference, here are the functions described in line 507 
[f1.pdf](https://github.com/Ouranosinc/xclim/files/9619010/f1.pdf)
[ratio.pdf](https://github.com/Ouranosinc/xclim/files/9619009/ratio.pdf)